gateway/src/main/java/io/camunda/zeebe/gateway/admin/backup/PartitionBackupStatus.java

```diff
@@ -10,14 +10,19 @@
 import io.camunda.zeebe.protocol.impl.encoding.BackupStatusResponse;
 import io.camunda.zeebe.protocol.management.BackupStatusCode;
 import java.util.Optional;
+import java.util.OptionalInt;
+import java.util.OptionalLong;
 
 public record PartitionBackupStatus(
```

Comment:
:wrench: Could we get rid of this type? If I understand correctly it's equivalent to the generated `PartitionBackupInfo` type.

Reply:
Same goes for `State` and `BackupStatus`. As far as I know, we only need two categories of types, the internal ones for the gateway <-> broker api and external ones that are generated from the OpenAPI spec.

Reply:
@oleschoenburg That is a valid point. But to reuse the generated classes, we should move the openapi spec from `dist` to `gateway`. The spec was added in dist because it is related to actuators which are implemented in `dist`. Should we move it to the gateway? My instinct is to keep in the dist, but I don't have a strong argument why it is better.

Reply:
Oh yeah, I didn't realize the generated types are not available here. It still feels unfortunate to have this intermediate type here but I agree with you that the generated types should stay in `dist`.